ui/src/hosts/utils/hostsSwitcherLinks.ts

```diff
@@ -0,0 +1,46 @@
+import {getAllHosts as getAllHostsAJAX} from 'src/hosts/apis'
+
+import {GetAllHosts} from 'src/types/apis/hosts'
+import {Source} from 'src/types/sources'
+import {HostNames, HostName} from 'src/types/hosts'
+import {DashboardSwitcherLinks} from 'src/types/dashboards'
+
+export const EMPTY_LINKS = {
+  links: [],
+  active: null,
+}
+
+const hostNamesAJAX = getAllHostsAJAX as GetAllHosts
```

Comment:
Try not to force typing here.  `const hostNamesAJAX: GetAllHosts = getAllHostsAJAX`